### PR TITLE
feat: Implement a new getPortalConfig service by site Id - EXO-65344 - Meeds-io/MIPs#72

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

File: component/portal/src/main/java/org/exoplatform/portal/mop/storage/SiteStorageImpl.java [hunk 1]
```diff
@@ -173,6 +173,16 @@ public PortalData getPortalConfig(PortalKey key) {
...
   }
 
+  @Override
+  public PortalData getPortalConfig(long siteId) {
+    SiteEntity entity = siteDAO.find(siteId);
+    if (entity != null) {
+      return buildPortalData(entity);
+    } else {
+      return null;
+    }
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/mop/storage/SiteStorageImpl.java]: you can use only on return here

**Review by azayati — CHANGES_REQUESTED** [review]
Change the title of the PR to describe only what it does

File: component/portal/src/main/java/org/exoplatform/portal/mop/storage/SiteStorageImpl.java [hunk 2]
```diff
@@ -173,6 +173,12 @@ public PortalData getPortalConfig(PortalKey key) {
     return getPortalConfig(siteKey);
   }
 
+  @Override
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/mop/storage/SiteStorageImpl.java]: is it possible to add a test into TestDataStorage.java ?

File: component/portal/src/main/java/org/exoplatform/portal/mop/service/LayoutServiceImpl.java [hunk 3]
```diff
@@ -117,6 +117,12 @@ public PortalConfig getPortalConfig(SiteKey siteKey) {
     return portalData == null ? null : new PortalConfig(portalData);
   }
 
+  @Override
+  public PortalConfig getPortalConfig(long siteId) {
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/mop/service/LayoutServiceImpl.java:121]: can you add an UT for that ?

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**